bindings/netstandard/ElectionGuard/ElectionGuard.Decryption/Decryption/CiphertextDecryptionTally.cs

```diff
@@ -65,14 +102,41 @@ public void AddBallot(CiphertextBallot ballot)
         {
             AddSpoiledBallot(ballot);
```

Comment:
So ballots are added as either challenged or spoiled only?  Not as just plain cast?

Reply:
correct. The cast ballots are added to the `CiphertextTally` (responsible for tally accumulation) along with Id's of the challenge/spoiled ballots. The `CiphertextDecryptionTally` (responsible for tally decryption) takes in the Ciphertext Tally (really just a big ballot representing all selections on all cast ballots) along with the spoiled/challenge ballots to do the decryption process. the spoiled ballots really arent mutated or used in any way (but they could be decrypted if they have to be, depending on the use case)